src/Version/Schema/ImageVersion.php

```diff
@@ -42,7 +42,7 @@ public function __construct(
         private readonly int $fileSize,
         #[Property(description: 'mime type', type: 'string', example: 'image/png')]
         private readonly string $mimeType,
-        #[Property(description: 'Metadata', type: 'array', items: new Items(ref: CustomMetadataVersion::class))]
+        #[Property(description: 'Metadata', type: 'array', items: new Items(ref: CustomMetaDataVersion::class))]
```

Comment:
Should this be also 'MetaData'?

Reply:
Yeah we should also change the description good catch!